modules/webgl2-polyfill/src/polyfill-context.js

```diff
@@ -10,7 +10,11 @@
 /* eslint-disable camelcase, brace-style */
 import assert from './assert';
 
-import {WEBGL2_CONTEXT_POLYFILLS, WEBGL2_CONTEXT_OVERRIDES} from './polyfill-table';
+import {
+  WEBGL2_CONTEXT_POLYFILLS,
+  // WEBGL2PLUS_CONTEXT_POLYFILLS,
```

Comment:
Should we support WebGL2+ polyfills here? This is pretty clearly a "WebGL3" (OpenGLES 3.x) API...